### PR TITLE
docs(examples): make it v12 compatible

### PR DESCRIPTION
**Pull request by subhod-i**
Fixed node examples.

### CODE REVIEW

**Comment by subhod-i** [comment]
Depends on PR #1567  #1565.

**Review by davidyuk — APPROVED** [review]
I think we can name `aeSdk` instances of both `AeSdkWallet` and `AeSdkAepp`, because it is not expected to use any other sdk class at the same time.

File: docs/contrib/README.md [hunk 1]
```diff
@@ -8,9 +8,6 @@ low-level access to it's endpoints, when necessary.
 
 It uses the following Javascript technologies and principles:
 
-- [stampit] provides composable Factories based on the [Stamp
-  Specification]. This is how aepp-sdk approached the [composition over
-  inheritance] principle.
```
Comment (davidyuk) [inline docs/contrib/README.md:13]: I don't think that we need to keep previously used technologies, someone can check git history if he need

**Comment by subhod-i** [comment]
> 



> I think we can name `aeSdk` instances of both `AeSdkWallet` and `AeSdkAepp`, because it is not expected to use any other sdk class at the same time.

ok then, it is finally **`aeSdk`**.

**Pull request merged**